lib/logger/lib/logger.ex

```diff
@@ -83,12 +104,14 @@ defmodule Logger do
   `config/config.exs`) but also changed dynamically during runtime via
   `Logger.configure/1`.
 
-    * `:level` - the logging level. Attempting to log any message
-      with severity less than the configured level will simply
-      cause the message to be ignored. Keep in mind that each backend
-      may have its specific level, too. Note that, unlike what happens with the
-      `:compile_time_purge_level` option, the argument passed to `Logger` calls
-      is evaluated even if the level of the call is lower than `:level`.
+    * `:level` - the logging level. Attempting to log any message with severity
+      less than the configured level will simply cause the message to be
+      ignored. Keep in mind that each backend may have its specific level,
+      too. Note that, unlike what happens with the `:compile_time_purge_level`
+      option, the argument passed to `Logger` calls is evaluated even if the
+      level of the call is lower than `:level`. For expensive to compute
+      messages it's a good idea to wrap them in a zero argument function, so the
+      computation only occurs when this option demands it.
```

Comment:
Please let's not wrap here. We don't have a convention of wrapping at a specific column and this block of text doesn't look to wide to me to justify rewrapping it completely.